<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>786</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="main_layout">
     <property name="spacing">
      <number>1</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item alignment="Qt::AlignRight">
      <widget class="HeaderWidget" name="tools_layout" native="true"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="mapGraphics_layout"/>
     </item>
     <item>
      <widget class="QTabWidget" name="tab_widget">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>150</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="log_layout">
        <attribute name="title">
         <string>Log</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTextBrowser" name="log_browser">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_layout">
        <attribute name="title">
         <string> Errors</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDockWidget" name="dockWidget">
     <property name="maximumSize">
      <size>
       <width>400</width>
       <height>524287</height>
      </size>
     </property>
     <property name="features">
      <set>QDockWidget::DockWidgetMovable</set>
     </property>
     <property name="windowTitle">
      <string/>
     </property>
     <widget class="QWidget" name="dockWidgetContents"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HeaderWidget</class>
   <extends>QWidget</extends>
   <header location="global">HeaderWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
